Makefile

```diff
@@ -0,0 +1,11 @@
+test:
+	pytest tests
+
+test_coverage:
+	COVERAGE_FILE=/tmp/.coverage pytest -vv --cov=merkle_zeppelin
+
+install_pre_commit_hooks:
+	pre-commit install -c .github/pre-commit-config.yaml
+
+update_pre_commit_hooks:
+	pre-commit install -c .github/pre-commit-config.yaml
```

Comment:
Is the update command really the same as the install one? I found [this](https://pre-commit.com/):
>Updating hooks automatically [¶](https://pre-commit.com/#updating-hooks-automatically)
>
>You can update your hooks to the latest version automatically by running [pre-commit autoupdate](https://pre-commit.com/#pre-commit-autoupdate). By default, this will bring the hooks to the latest tag on the default branch.

Reply:
that's strange, probably intentional copy-paste
thanks!